test/replication.test.js

```diff
@@ -1921,99 +1919,105 @@ describe('Replication / Change APIs with custom change properties', function() {
     dataSource = this.dataSource = loopback.createDataSource({
       connector: loopback.Memory,
     });
-    SourceModelWithCustomChangeProperties =
-    this.SourceModelWithCustomChangeProperties = PersistedModel.extend(
+    SourceModel = this.SourceModel = PersistedModel.extend(
       'SourceModelWithCustomChangeProperties-' + tid,
-      {id: {id: true, type: String, defaultFn: 'guid'}, customProperty: {type: 'string'}},
-      {trackChanges: true, additionalChangeModelProperties: {customProperty: {type: 'string'}}});
+      {
+        id: {id: true, type: String, defaultFn: 'guid'},
+        customProperty: {type: 'string'},
+      },
+      {
+        trackChanges: true,
+        additionalChangeModelProperties: {customProperty: {type: 'string'}},
+      });
+
+    SourceModel.createChangeFilter = function(since, modelFilter) {
```

Comment:
Moved the setup of `createChangeFilter` and `fillCustomChangeProperties` to the before-each hook, as it's tied to `additionalChangeModelProperties` that's configured in this hook too.